CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Features
 - Add a "docs" field to models, with a "show" subfield ([#1671](https://github.com/fishtown-analytics/dbt/issues/1671), [#2107](https://github.com/fishtown-analytics/dbt/pull/2107))
+- Add support for generating database name macro ([#1695](https://github.com/fishtown-analytics/dbt/issues/1695), [#2143](https://github.com/fishtown-analytics/dbt/pull/2143))
```

Comment:
Can we add a separate entry for the breaking change that 1-arg `generate_schema_name` macros are no longer supported?